code/modules/mob/living/carbon/human/species.dm

```diff
@@ -965,7 +965,7 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 		H.losebreath = 0
 
 		var/takes_crit_damage = (!HAS_TRAIT(H, TRAIT_NOCRITDAMAGE))
-		if((H.health < H.crit_threshold) && takes_crit_damage)
+		if((H.health < H.crit_threshold) && takes_crit_damage && H.stat != DEAD)
```

Comment:
but spec_life shouldn't be called when `H.stat == DEAD`. This is a fix in the wrong place.

Reply:
I tested it locally with an without the change, somehow, it works

Reply:
this will cause issues with shadowpeople and maybe some other species, the old solution is fine

Reply:
Reset to original commit as per Fikou's comment

Reply:
Wha